### PR TITLE
B-18326 Update the Estimated Weight - B-18327 Remove Fields - B-18322 Update Requested pickup date

### PR DESCRIPTION
**Pull request by WeatherfordAaron**
[Previous PR into integration testing](https://github.com/transcom/mymove/pull/11803)

### CODE REVIEW

**Review by KonstanceH — COMMENTED** [review]
- [ ] There are a couple floating ';' on the page
Other than that everything looks good to me

**Comment by taeJungCaci** [comment]
Changes in this PR doesn't seem to match the PR to Integration.

For example:

 `columnHeaders={[headerPlannedMoveDate, headerActualMoveDate, headerActualArrivalDate]}`

is in the PR to Integration for not on this one.

**Comment by JamesHawks224** [comment]
test issue:
";" artifact showing on screen

image

**Comment by JamesHawks224** [comment]
test issue:

test step 4 failed.
Observe that none of the follow fields show up:
...
-Requested Pickup Date
-Scheduled Pickup Date
-Actual Pickup Date

These fields show up on the MTO page
image

**Comment by JamesHawks224** [comment]
testing issue:

Step 4.
The following dates were enetered during shipment setup.
orders date:  1/3/2024
rpt by date:  4/2/2024
planned move date:  1/16/2024

No values show up in the MTO
image

**Review by taeJungCaci — CHANGES_REQUESTED** [review]
https://github.com/transcom/mymove/pull/11941#issuecomment-1934431587

**Comment by WeatherfordAaron** [comment]
> Changes in this PR doesn't seem to match the PR to Integration.
> 
> For example:
> 
> `columnHeaders={[headerPlannedMoveDate, headerActualMoveDate, headerActualArrivalDate]}`
> 
> is in the PR to Integration but not on this one.

I checked the old branch against this branch and am not seeing a diff. Can you check one more time?

**Comment by JamesHawks224** [comment]
Verification rpt attached.
There were differences.
[B18326.verify_rpt.txt](https://github.com/transcom/mymove/files/14377372/B18326.verify_rpt.txt)

**Comment by taeJungCaci** [comment]
> > Changes in this PR doesn't seem to match the PR to Integration.
> > For example:
> > `columnHeaders={[headerPlannedMoveDate, headerActualMoveDate, headerActualArrivalDate]}`
> > is in the PR to Integration but not on this one.
> 
> I checked the old branch against this branch and am not seeing a diff. Can you check one more time?

This was discussed through a DM in slack. Adding this comment as a reference.

**Comment by JamesHawks224** [comment]
Per dev, main pr is correct and complete.

**Review by KonstanceH — APPROVED** [review]

File: src/components/Office/ShipmentDetails/ShipmentDetailsMain.jsx [hunk 1]
```diff
@@ -158,6 +158,9 @@ const ShipmentDetailsMain = ({
       displayedDeliveryAddress = destinationAddress;
       break;
     case SHIPMENT_OPTIONS.PPM:
+      weightResult = ppmShipment.estimatedWeight;
+      displayedPickupAddress = pickupAddress;
+      displayedDeliveryAddress = destinationAddress || destinationDutyLocationAddress;
```
Comment (taeJungCaci) [inline src/components/Office/ShipmentDetails/ShipmentDetailsMain.jsx:163]: Looks like you have duplicated changes. These lines are already in line 166~168

**Comment by WeatherfordAaron** [comment]
> Changes in this PR doesn't seem to match the PR to Integration.
> 
> For example:
> 
> `columnHeaders={[headerPlannedMoveDate, headerActualMoveDate, headerActualArrivalDate]}`
> 
> is in the PR to Integration but not on this one.

The current state of the ask from the PO is not to have the 3 fields but rather to only have 2. The names are correct at this time AFAIK.

File: src/components/Office/ShipmentDetails/ShipmentDetailsMain.jsx (hunk 1 above)
Comment (WeatherfordAaron) [reply src/components/Office/ShipmentDetails/ShipmentDetailsMain.jsx]: Pushing that fix now. Thank you buddy

Comment (taeJungCaci) [reply src/components/Office/ShipmentDetails/ShipmentDetailsMain.jsx]: No problem! But if those 3 lines of code are all there is in this PR, isn't that mean the changes are already in main and this PR is actually not needed?

Comment (taeJungCaci) [reply src/components/Office/ShipmentDetails/ShipmentDetailsMain.jsx]: because I see 0 files changed now in this PR

Comment (WeatherfordAaron) [reply src/components/Office/ShipmentDetails/ShipmentDetailsMain.jsx]: Correct

**Pull request closed without merging**